features/support/selectors.rb

```diff
@@ -0,0 +1,44 @@
+# TL;DR: YOU SHOULD DELETE THIS FILE
```

Comment:
Do you need this file?

Reply:
I tried removing it and the cucumber tests fail to build so I restored it back.